sdk/storage/Azure.Storage.Blobs/src/BlobBaseClient.cs

```diff
@@ -1032,6 +1032,7 @@ private void SetNameFieldsIfNull()
         ///     </item>
         /// </list>
         /// </remarks>
+        [EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
Why are we hiding `DownloadStreaming()`?

Reply:
These should have been hidden already. This is the only method we did not remove the options bag from, as it got another feature elsewhere. This corrects that mistake, leaving us with a sync/async pair of options bag overloads visible and this sync/async pair hidden. The current state follows the pattern we want to follow.

Reply:
Resolved offline that this is the correct state.